packages/salesforcedx-utils-vscode/test/unit/cli/deployResultParser.test.ts

```diff
@@ -223,4 +223,37 @@ describe('force:source:deploy parser', () => {
       throw Error('Successes should be present but were not returned');
     }
   });
+
+  it('Should parse source:push success successfully', () => {
+    const response = {
+      status: 0,
+      result: {
+        pushedSource: [{
+          state: 'Add',
+          type: 'ApexClass',
+          fullName: 'MyClass',
+          filePath: 'src/classes/MyClass.cls'
+        }]
+      }
+    };
+
+    const parser = new ForceDeployResultParser(JSON.stringify(response));
+    const successes = parser.getSuccesses();
+    if (successes) {
```

Comment:
Could you add an else case to handle no successes?

Reply:
Ah yes good catch, thanks!